package NGenerativeBoltalka;

enum ETokenizerType {
    GENERATIVE = 0;
    ZELIBOBA = 1;
    ZELIBOBA_SEP = 2;
    ZELIBOBA_PREFIX = 3;
    ZELIBOBA_SEP_NAMES = 4;
    ZELIBOBA_EXTERNAL_INFO = 5;
};

enum ETruncatePolicy {
    KeepSuffix = 0;
    KeepPrefix = 1;
}